Swiftilities.podspec

```diff
@@ -12,7 +12,12 @@ Pod::Spec.new do |s|
 
   s.homepage         = "https://github.com/rightpoint/Swiftilities"
   s.license          = 'MIT'
-  s.author           = { "Nicholas Bonatsakis" => "nick.bonatsakis@raizlabs.com" }
+  s.authors          = {
+                         "Chris Ballinger" => "chris.ballinger@raizlabs.com",
+                         "Nicholas Bonatsakis" => "nick.bonatsakis@raizlabs.com",
+                         "Zev Eisenberg" => "zev.eisenberg@raizlabs.com",
```

Comment:
Nick’s Raizlabs address is probably dead. These should all be Rightpoint, and Nick should either be removed or have an updated address

Reply:
+1 Good call.

Reply:
Done!